test/integration/serverside.js

```diff
@@ -604,7 +604,7 @@ describe('Managing users', function () {
 
 	it('ban user', async function () {
 		await client.banUser(evilUser, {
-			user_id: user.id,
+			banned_by_id: user.id,
```

Comment:
it might be a good idea to add an extra test to make sure the old field is still supported

```js
await client.post(client.baseURL + '/moderation/ban', {
      target_user_id: evilUser,
      user_id: user.id
 });

```

Reply:
It is done already, please see https://github.com/GetStream/stream-chat-js/pull/502/files#diff-d94fa608cd42452bb60618acc150397dd1e93ad02319fb4850c09cdcd32c2967R410

Reply:
these two sends a similar request to the API, code snippet above ensure API continues supporting deprecated field

Reply:
Ah, yes, got it.